manifests/base/generate_kustomization.py

```
@@ -0,0 +1,426 @@
+    elif args.stdout:
+        sys.stdout.write(content)
+    else:
+        OUTPUT_FILE.write_text(content)
+        print(f"Wrote {OUTPUT_FILE}")
+
+
+def _print_first_difference(existing: str, generated: str) -> None:
+    """Print a diagnostic showing the first line that differs."""
+    existing_lines = existing.splitlines()
+    generated_lines = generated.splitlines()
+    for i, (e, g) in enumerate(zip(existing_lines, generated_lines), 1):
```

Ruff (0.15.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)